var/spack/repos/builtin/packages/openmpi/package.py

```diff
@@ -178,11 +179,13 @@ class Openmpi(AutotoolsPackage):
     patch('configure.patch', when="@1.10.1")
     patch('fix_multidef_pmi_class.patch', when="@2.0.0:2.0.1")
 
+    fabrics = ('psm', 'psm2', 'verbs', 'mxm', 'ucx', 'libfabric')
```

Comment:
adds support for libfabric and ucx and makes fabrics a variable that can be used in when decorators below